development-support.rst

```diff
@@ -0,0 +1,16 @@
+.. _application-development-support:
+
+
+Application development support
+================================
+
+Divio's global network of trusted partner agencies is able to help companies of all sized, international organisations
+and academic institutions bring even the largest and most complex projects smoothly to fruition. 
+
+Our partners can be found on almost every continent. They have a vast range of web development experience and expertise
+in project implementation on the Divio.
+
+Whatever your requirements, the partnership network has the skills and experience to help you accomplish your aims on
+the web. 
+
+Find a Divio partner for your project `here <https://www.divio.com/company/partners/>`_.
```

Comment:
Never use the word "here" as the text for a link (this is bad for accessibility, see e.g. https://webaccess.berkeley.edu/ask-pecan/click-here). 

A correct form might be: 

```
`Find a Divio Partner <https://www.divio.com/company/partners/>`_.

Reply:
Done